db/migrate/20230403150603_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+    rename_column :posts, :user_id, :author_id
+  end
+end
```

Comment:
- [ ] Associations between models must be configured correctly according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Great job so far 👏🏻 however, since we are using the `author_id` column to reference the modal user, we need to show that the `'author_id'` references the User class. to do this, you need to update the value of the 'foreign_key' attribute to `{ to_table: :users }`. You can write the query like this: 


```ruby
class AddUserRefToPosts < ActiveRecord::Migration[7.0]
  def change
    add_reference :posts, :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
  end
end
```